### ISSUE TITLE
About code font size.

### ISSUE THREAD

**Issue by gapplef** [theme-next/hexo-theme-next#687]
### I agree and want to create new issue 

- [ ] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

According to my understand, current NexT theme set the code font size(`$code-font-size`) to a fix value `14px`, while the font of article body is set  to changes according to the screen size(-4px, -2px, +2px, +4px), with base font(`$font-size-base`) also set to `14px`.
https://github.com/theme-next/hexo-theme-next/blob/dcf789dc7b7ebb33e68738a43accafd68970b6bd/source/css/_variables/base.styl#L137-L138

https://github.com/theme-next/hexo-theme-next/blob/dcf789dc7b7ebb33e68738a43accafd68970b6bd/source/css/_variables/base.styl#L87-L92

This means the font size of code could be same with, smaller or larger than the article body font in different case. I don't think this is appropriate, especially for the inline code.

It's very common to use  inline code to highlight `keywords`, like in the [office documentation](https://theme-next.org/docs/faqs/index.html). I think size of  these keywords should be same with the article body.

To sum up,  I would like the font size of code always same with the article body, especially for inline code. How to config the NexT theme?

### NexT Information

**NexT Version:**

- [ ] Latest Master branch
- [x] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [x] All schemes
- [ ] Muse
- [ ] Mist
- [ ] Pisces
- [ ] Gemini

**Comment by stevenjoezhang** [comment]
Temporary solution: delete this line
https://github.com/theme-next/hexo-theme-next/blob/dcf789dc7b7ebb33e68738a43accafd68970b6bd/source/css/_common/components/highlight/highlight.styl#L25

**Comment by gapplef** [comment]
@stevenjoezhang   here seems miss `:` ?
https://github.com/theme-next/hexo-theme-next/blob/dcf789dc7b7ebb33e68738a43accafd68970b6bd/source/css/_common/components/highlight/highlight.styl#L11

**Comment by stevenjoezhang** [comment]
Emm... I'll open a PR to fix it...

**Comment by gapplef** [comment]
Great jobs! Appreciate for your quick response.

**Comment by stevenjoezhang** [comment]
Fixed in #1005, #1006
